src/app/content/utils/highlightSharedUtils.ts

```diff
@@ -0,0 +1,7 @@
+import mapValues from 'lodash/fp/mapValues';
+import pickBy from 'lodash/fp/pickBy';
+import { isDefined } from '../../guards';
+import { CountsPerSource } from '../types';
+
+export const extractTotalCounts = (countsPerSource: CountsPerSource) =>
```

Comment:
again, its cool that this is here, but you're still reaching into highlights for a bunch of other utils, we need to either go one way or the other